src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -359,6 +377,7 @@ bool collectionUpdate
   {
     connection->update(col.c_str(), q, doc, upsert);
     releaseMongoConnection(connection);
+    TIME_STAT_MONGO_WRITE_WAIT_STOP();
```

Comment:
I'll try to clarify.

Let's imagine a given operation at mongoBackend that includes 1 query operation 2 cmd operations and 5 write operations. In that case, after processing the request, what would be set in "last" and accumulated in "acc" for mongoWriteTime? The time corresponding to the sum of the 5 write operations? Or the time of the fifth write operation?